_partials/ubuntu_python.md

```diff
@@ -6,6 +6,7 @@ Lets install pyenv to manage our python versions:
 
 ```bash
 git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+source ~/.zprofile
```

Comment:
Before this edit, had to add the following in my .zshrc to make pyenv work
```
export PYENV_ROOT="$HOME/.pyenv"
export PATH="$PYENV_ROOT/bin:$PATH"
```

Reply:
`.zprofile` is only loaded at VM startup, while zshrc is loaded at teach shell startup.

`source ~/.zprofile` is just to avoid you restarting the VM for this setup 

You can safely remote it from you rzshrc now